### ISSUE TITLE
Orden

### ISSUE THREAD

**Issue by ivovila** [PI-ITBA/TPE#25]
Hola Marcelo, te quería consultar si podemos utilizar un algoritmo de ordenación encontrado en internet, como el heap sort o bubble sort, ya que no lo vimos en clase, pero fue utilizado en una practica.

**Comment by marcelogarberoglio** [comment]
Sí, pueden usarlo. Si necesitan usar vectores y ordenarlos pueden usar la función qsort de la biblioeca standard
